### PR TITLE
Documentation: Doc/update python environment if getting new Ansys version

### PR DESCRIPTION
**Pull request by anslpa**

### CODE REVIEW

File: docs/source/getting_started/compatibility.rst [hunk 1]
```diff
@@ -61,6 +61,23 @@ should also be synchronized with the server version.
      - 0.2.*
 
 
+Updating Python environment
+---------------------------
+
+Passing from an Ansys version to another (e.g. passing from Ansys 2022 R2 to Ansys 2023 R1), it is needed
```
Comment (anslpa) [inline docs/source/getting_started/compatibility.rst:67]: Updating ?

File: docs/source/getting_started/compatibility.rst [hunk 2]
```diff
@@ -61,6 +61,23 @@ should also be synchronized with the server version.
      - 0.2.*
 
 
+Updating Python environment
+---------------------------
+
```
Comment (cbellot000) [inline docs/source/getting_started/compatibility.rst:66]: Passing to upgrading maybe?

File: docs/source/getting_started/compatibility.rst (hunk 1 above)
Comment (PipKat) [reply docs/source/getting_started/compatibility.rst:67]: @anslpa The Google Developer Style Guidelines say not to use Latin phrases, such as e.g. or i.e.). I'm not exactly sure what you want to say. When moving from one Ansys release to another, you must update the ``ansys-dpf-core`` package and its dependencies. To get the latest version of the ``ansys-dpf-core`` package, use this code:

File: docs/source/getting_started/compatibility.rst [hunk 3]
```diff
@@ -61,6 +61,23 @@ should also be synchronized with the server version.
...
+Passing from an Ansys version to another (e.g. passing from Ansys 2022 R2 to Ansys 2023 R1), it is needed 
+to update ansys-dpf-core package and its dependencies. To get the latest version of ansys-dpf-core, use:
+
+.. code::
+    
+	pip install --upgrade --force-reinstall ansys-dpf-core
+
+To get a specific version of ansys-dpf-core (e.g. 0.7.0), use:
```
Comment (PipKat) [inline docs/source/getting_started/compatibility.rst:74]:
```suggestion
To get a specific version of the ``ansys-dpf-core`` package, such as 0.7.0, use this code:
```

File: docs/source/getting_started/compatibility.rst (hunk 1 above)
Comment (anslpa) [reply docs/source/getting_started/compatibility.rst:67]: Thanks !

**Review by PProfizi — APPROVED** [review]

**Pull request merged**